Tests/js/views/tabs/assets/ez-locationviewdetailstabview-tests.js

```diff
@@ -584,10 +598,69 @@ YUI.add('ez-locationviewdetailstabview-tests', function (Y) {
         },
     });
 
+    fireLoadSectionEventTest = new Y.Test.Case({
+        name: "eZ LocationViewDetailsTabView fire load section event test",
+        setUp: function () {
+            _eventTestSetup(this);
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should fire the loadSection event when view become active": function () {
+            var that = this,
+                sectionCalled = false;
+
+            this.view.on('loadSection', function (e) {
+                sectionCalled = true;
+                Assert.areSame(
+                    that.view.get('content'),
+                    e.content,
+                    "The event facade should contain the content"
+                );
+            });
+
+            this.view.set('active', true);
+
+            Assert.isTrue(sectionCalled, "loadSection should have been called");
+        },
+
+        "Should try to reload the content when tapping on the retry button": function () {
+            var loadSection = false;
+
+            this.view.render();
+            this.view.set('active', true);
+            this.view.set('loadingError', true);
+
+            this.view.on('loadSection', function () {
+                loadSection = true;
+            });
+
+            this.view.get('container').one('.ez-asynchronousview-retry').simulateGesture('tap', this.next(Y.bind(function () {
```

Comment:
`Y.bind` call is useless, the second parameter of `next` is the context in which the function will be executed.